cmd/kubeadm/app/cmd/defaults.go

```diff
@@ -29,8 +29,7 @@ import (
 )
 
 var (
-	// Maximum version when using AllowAll as the default authz mode. Everything above this will use RBAC by default.
-	allowAllMaxVersion = semver.MustParse("1.6.0-alpha.0")
+	minK8sVersion = semver.MustParse("1.6.0-alpha.1")
```

Comment:
Ah, ok. Yeah, makes sense. I can do that in a follow up, I've already rebased upon this commit in my larger PR